src/lib.rs

```diff
@@ -1,7 +1,102 @@
 #![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
 
 #[cfg(not(windows))]
+#[cfg(not(use_rust_bindgen))]
 include!("bindings_macos.rs");
 
 #[cfg(windows)]
-include!("bindings_windows.rs");
\ No newline at end of file
+#[cfg(not(use_rust_bindgen))]
+include!("bindings_windows.rs");
+
+#[cfg(use_rust_bindgen)]
+include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+mod helper;
+
+// MySQL 8.0.1 Remove my_bool
+// https://bugs.mysql.com/bug.php?id=85131
+#[cfg(MySql_Version_AboveOrEqual_8_0_1)]
```

Comment:
We can and likely should use this cfg to conditionally include different bindings based on the version if the bindgen flag is not set.